vmcnet/train/parse_config_flags.py

```diff
@@ -25,10 +25,18 @@ def _get_config_from_reload(
     return flag_values.config
 
 
-def _get_config_from_default_config(flag_values: flags.FlagValues) -> ConfigDict:
+def _get_config_from_default_config(
+    flag_values: flags.FlagValues, *presets_paths
+) -> ConfigDict:
+    base_config = train.default_config.get_default_config()
+
+    for path in presets_paths:
```

Comment:
Why are there multiple preset_paths? It looks like you only pass a single one below and I'm not sure if more would make sense?

Reply:
I was wondering about the possibility of having the default settings given as default.json instead of a python function so that presets_paths would be (default.json, my_preset.json)

Reply:
changed to single preset paths